.github/renovate.json5

```diff
@@ -1,13 +1,13 @@
 {
   repositories: ['simple-icons/simple-icons'],
   extends: [
-    'config:base',
+    'config:recommended',
 
     // Make sure we get a single PR combining all updates
     'group:all',
   ],
 
-  // Disable dependency dashboard
+  // Disable Dependency Dashboard
   dependencyDashboard: false,
```

Comment:
_Why_ you're disabling the Dependency Dashboard? I generally like having the Dependency Dashboard. :smile: 

Read the [Renovate docs, key concepts, Dependency Dashboard](https://docs.renovatebot.com/key-concepts/dashboard/) page to learn what the Dependency Dashboard is about, and how it can help.

Reply:
Can you explain what are the real advantadges of the dashboard in this repository? Note that we do updates with our Simple Icons bot self-hosted in a GH workflow that runs twice a year.

Reply:
## Benefits of Dependency Dashboard

With the Dependency Dashboard you can:

- See if you have updates for your dependencies
- Opt-in to Dependency Dashboard Approval workflow for some dependencies that you want to update semi-manually
- See a summary of Renovate PRs that you have closed before, maybe due to some upgrade conflict or other thing you needed to do first, before you can pull in the update
- Get updates "ahead of the configured schedule" when you want

## Run the Renovate program more often

https://github.com/simple-icons/simple-icons/blob/58faa90925a900b61fb7a3feb96c6028cf483607/.github/workflows/renovate.yml#L3-L5

You are running the Renovate program only once every six months. I do _not_ recommend this.

You must give Renovate enough time each day to do useful things for you. If you find Renovate too noisy, the _proper_ fix is to improve your Renovate config file, not in limiting physical runtime for the bot.

### If you find Renovate too noisy there's ways to fix that in config. :wink: 

Please read the [Renovate docs, Noise Reduction](https://docs.renovatebot.com/noise-reduction/) for the full info.

To summarize, and in broad strokes, some ways to reduce noise:

- Group packages that need to updated together in one PR
- Schedule when Renovate is allowed to create updates for certain packages, or even for all your packages
- Renovate always create security critical updates at once, provided you're actually running the bot often enough
- You may decide to let Renovate automerge certain dependencies when the test suite passes
- If you use GitHub Merge Queue, Renovate can work with that as well, to speed up merges, and to reduce review workload
- You can opt-in to the [Dependency Dashboard Approval workflow](https://docs.renovatebot.com/key-concepts/dashboard/#dependency-dashboard-approval-workflow), where you only get updates from Renovate after selecting them from the Dependency Dashboard

## What do you want out of the bot?

As you're limiting the physical run time of the bot to _two times a year_, I assume you want low-noise, high-signal out of the bot?

Reply:
> You are running the Renovate program only once every six months. I do not recommend this.

OK, but we've limited time and this project does not ship code that relies on dependencies. We update dependencies once every 6 months just 1 month before major releases.

> You must give Renovate enough time each day to do useful things for you. If you find Renovate too noisy, the proper fix is to improve your Renovate config file, not in limiting physical runtime for the bot.

This seems like a copy-pasted response. We don't run the bot, we run the program with GH actions. We previously tried to use the bot to get a grouped update several times a month but that resulted in the updates not being trigerred because the Renovate bot was too busy at that moment, which is a no-go.

Reply:
## Limited time

> OK, but we've limited time and this project does not ship code that relies on dependencies. We update dependencies once every 6 months just 1 month before major releases.

Because you have limited time, you are best helped with a bot that can do useful things for you. Let the bot worry about the updates, so you can focus mostly on your features and bugfixes.

[Renovate docs, in repository schedule configuration](https://docs.renovatebot.com/key-concepts/scheduling/#in-repository-schedule-configuration) says:

> ### In repository schedule configuration
> 
> Reminder: the times when the Renovate process runs are controlled by the bot admin using tools such as `cron`.
> If you use the Mend Renovate App, the default is that Renovate will always be allowed to run.
> 
> Be sure to schedule enough time for Renovate to process your repository.
> Do not set schedules like "Run Renovate for an hour each Sunday" as you _will_ run into problems.
> 
> Some config examples:
> 
> ```json title="Renovate should run each day before 4 am"
> {
>   "schedule": ["before 4am"]
> }
> ```
> 
> ```json title="Renovate should run outside of common office hours"
> {
>   "schedule": [
>     "after 10pm every weekday",
>     "before 5am every weekday",
>     "every weekend"
>   ]
> }
> ```

Again, limiting the physical bot runtime on GitHub Actions is the wrong way to schedule _when_ Renovate takes actions on your repository.

## Copy/paste response?

> This seems like a copy-pasted response. We don't run the bot, we run the program with GH actions. We previously tried to use the bot to get grouped updates several times a month but that resulted in the updates not being trigerred because the Renovate bot was too busy at that moment, which is a no-go.

- That was a fresh comment, just for this project. Ofcoure the general theme matches the docs recommendations.
- Renovate is _not_ too busy, it's actually not allowed to run often enough in GitHub Actions to check for updates, and to create update PRs.

## Where to get configuration help

If you want to improve your Renovate configuration, and want help, go to the [Renovate repository, discussions](https://github.com/renovatebot/renovate/discussions) and open a new "Request help" Discussion. We'll be happy to help you improve the configuration.

I'll stop talking about this now, and let you decide what to do. 😉